test/example/qt.erl

```diff
@@ -0,0 +1,402 @@
+%% CTL-C on both the server and the client.
+
+
+
+%% The server
+
+-define(INTERVAL, 3000).
+
+s() ->
+    application:ensure_all_started(quicer),
+    Port = 4567,
+    LOptions = [ {cert, "cert.pem"}
+               , {key,  "key.pem"}
+               , {handshake_idle_timeout_ms, 3 * ?INTERVAL}
+               , {keep_alive_interval_ms, ?INTERVAL}
+               , {alpn, ["sample"]}
+               , {idle_timeout_ms, 3 *?INTERVAL}
+               , {peer_bidi_stream_count, 64000}
+               ],
+    proc_lib:spawn_link(fun() ->
+                                {ok, L} = quicer:listen(Port, LOptions),
+                                listener(L)
+                        end).
+
+
+listener(L) ->
+    Top = self(),
+    P = proc_lib:spawn_link(
+          fun() ->
+                  acceptor(Top, L)
+          end),
+    receive {P, more} -> ok end,
+    listener(L).
+
+
+acceptor(Top, L) ->
+    io:format("Call accept ~p\n", [self()]),
+    {ok, Conn} = quicer:accept(L, [], infinity),
+
+    Top ! {self(), more},
+
+    io:format("accept -> ~p\n",[Conn]),
+
+    case quicer:handshake(Conn) of
+        {ok, Conn} ->
+            io:format("Handshake ok\n",[]),
+            accept_stream_loop(Conn);
+        Err ->
+            io:format("Failed handshake ~p~n", [Err])
+    end.
+
+
+
+accept_stream_loop(Conn) ->
+    case quicer:accept_stream(Conn, [{active, false}], infinity) of
+        {ok, Stm} ->
+            io:format("accept stream -> ~p\n",[Stm]),
+
+            Self = self(),
+            H = proc_lib:spawn_link(
+                  fun() -> stream_owner(Self, Stm) end),
+            ok = quicer:controlling_process(Stm, H),
+
+            receive
+                {H, continue} ->
+                    accept_stream_loop(Conn)
+            end;
+        {error, connection_closed} ->
+            io:format("Connection closed \n",[]),
+            ok;
+        Err ->
+            io:format("Failed to accept_stream ~p \n",[Err]),
+            accept_stream_loop(Conn)
+    end.
+
+stream_owner(Top, Stream) ->
+    Top ! {self(), continue},
+    case server_negotiate(Stream) of
+        {ok, CNo, SNo} ->
+            io:format("Enter recv_ping for ~p:~p\n",[CNo, SNo]),
+
+            ok = quicer:setopt(Stream, active, 20),
+            recv_ping(Stream, 1, CNo, SNo),
+            io:format("Leaving stream_Owner \n",[]);
+        Err ->
+            io:format("server negotiate ~p~n", [Err])
+    end.
+
+
+
+recv_ping(S, N, CNo, SNo) ->
+    receive
+        {quic_passive, S} ->
+            ok = quicer:setopt(S, active, 20),
+            recv_ping(S, N, CNo, SNo);
+        {quic, <<"ping">>, S,_} ->
+            io:format("Got ping ~p from ~p:~p\n",[N, CNo, SNo]),
+            {ok, 4} = quicer:send(S, <<"pong">>),
+            recv_ping(S, N+1, CNo, SNo);
+        {quic, <<"die">>, S,_} ->
+            io:format("Got die from ~p:~p\n",[CNo, SNo]),
+            die;
+        {quic, peer_send_shutdown, S} ->
```

Comment:
In latest main branch, we use 4 elements tuple.  so should be 
```
{quic, peer_send_shutdown, S, _} ->
```

Reply:
Yes, indeed, fixed, pushed